CHANGELOG/CHANGELOG-2.8.0.md

```diff
@@ -0,0 +1,95 @@
+# Release notes for v2.8.0
+
+[Documentation](https://kubernetes-csi.github.io/docs/)
+
+# Changelog since v2.7.0
+
+## Changes by Kind
+
+### Other (Cleanup or Flake)
+
+- Fix: CVE-2021-38561 in image build (#148, @andyzhangx)
+- Bump kubernetes to v1.25, CSI Spec to v1.6 (#154, @humblec)
```

Comment:
Can you change (#154, @humblec) to ([#154](https://github.com/kubernetes-csi/livenessprobe/pull/154),[ @humblec](https://github.com/humblec))?